src/_pytest/debugging.py

```diff
@@ -256,7 +256,7 @@ def _test_pytest_function(pyfuncitem):
     _pdb = pytestPDB._init_pdb()
     testfunction = pyfuncitem.obj
     pyfuncitem.obj = _pdb.runcall
-    if "func" in pyfuncitem._fixtureinfo.argnames:  # noqa
+    if "func" in pyfuncitem._fixtureinfo.argnames:
```

Comment:
@nicoddemus 
This was from you.. am I missing something?

Reply:
Changed to "# pragma …" - when quickly checking before I've thought the whole function wasn't covered.

Was "# noqa" actually picked up?